### PR TITLE
os/bluestore: a bunch of minor fixes

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ifed01** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-10-13_01:10:32-rados-wip-kefu-testing-2020-10-12-1615-distro-basic-smithi
- https://pulpito.ceph.com/kchai-2020-10-14_11:01:53-rados-wip-kefu-testing-2020-10-14-1541-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-10-15_08:32:19-rados-wip-kefu-testing-2020-10-14-1541-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-10-15_10:09:42-rados-wip-kefu-testing-2020-10-14-1541-distro-basic-smithi/

**Comment by yuriw** [comment]
[wip-yuri-testing-2020-10-26-0817](https://trello.com/c/1Fh3DURx/1047-wip-yuri-testing-2020-10-26-0817)

**Review by neha-ojha — APPROVED** [review]

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -13786,7 +13786,7 @@ void BlueStore::_wctx_finish(
     auto& r = lo.r;
     txc->statfs_delta.stored() -= lo.e.length;
     if (!r.empty()) {
-      dout(20) << __func__ << "  blob release " << r << dendl;
+      dout(20) << __func__ << "  blob1 " << *b << " release " << r << dendl;
```
Comment (tchaikov) [inline src/os/bluestore/BlueStore.cc:13789]: @ifed01 i understand that you want to print out `*b`, but why s/blob/blob1/ ?

Comment (ifed01) [reply src/os/bluestore/BlueStore.cc]: @tchaikov - just a typo. Fixed!

**Review by tchaikov — APPROVED** [review]

**Pull request merged**